bench/wtperf/wtperf.h

```diff
@@ -190,6 +199,7 @@ struct __config {			/* Configuration structure */
 #define	ELEMENTS(a)	(sizeof(a) / sizeof(a[0]))
 
 #define	THROTTLE_OPS	100
+#define	THROTTLE_INTVL	(USEC_PER_SEC / THROTTLE_OPS)
```

Comment:
The names of these macros does not make sense to me with their usage.  Both are used in various ways for both operation counts and time-related calculations.  I'd rather see more, clearer macros that may use the same values as others.  However, I think I may have come to the conclusion that `THROTTLE_INTVL` is not needed.